frontend/Gruntfile.js

```diff
@@ -438,7 +443,7 @@ module.exports = function (grunt) {
     grunt.registerTask('default', ['jshint:all', 'less', 'copy:local-all', 'copy:local-index']);
     grunt.registerTask('baseDEV', ['handlebars:all', 'less', 'copy:all', 'processhtml:dev', 'copy:less', 'copy:config', 'copy:locales', 'concurrent:dev']);
     grunt.registerTask('baseDEMO', ['mkdir:demo', 'handlebars:all', 'less', 'copy:demo', 'processhtml:dev', 'copy:config', 'copy:locales']);
-    grunt.registerTask('baseBUILD', ['jsdoc', 'handlebars:temp', 'less', 'copy:build', 'processhtml:build', 'copy:config', 'copy:locales', 'copy:temp', 'requirejs', 'uglify']);
+    grunt.registerTask('baseBUILD', ['jsdoc', 'handlebars:temp', 'less', 'copy:build', 'processhtml:build', 'copy:config','copy:config-build', 'copy:locales', 'copy:temp', 'requirejs', 'uglify']);
```

Comment:
Do you really need both, `copy:config` **and** `copy:config-build`, btw? :thinking:

Reply:
The first copy the profile to the target workspace meanwhile the second you need for the `requirejs` task

Reply:
Yes but I would think that the former is not necessary since the RequireJS task should include the config file in the resulting bundle. Could you verify that it works without and then change it?

Reply:
Note also that there is a space missing here. :wink: